server/db/schema.ts

```diff
@@ -73,6 +73,16 @@ export type PostingApplication = typeof postingApplicantsTable.$inferSelect;
 
 //---------------**************----------------
 
+export const reviewTagsTable = pgTable('review_tags', {
+  id: serial('id').primaryKey(),
```

Comment:
wrap it like `defaultUuidPkField`, should be called `defaultSerialPkField`

Reply:
okay